examples/config-sync.json

```diff
@@ -0,0 +1,52 @@
+{
+	"version":"0.1.0-dev",
+	"storage":{
+		"rootDirectory":"/tmp/zot"
+	},
+	"http":{
+		"address":"127.0.0.1",
+		"port":"5000"
+	},
+	"log":{
+		"level":"debug"
+	},
+	"extensions":{
+		"sync":[
+			{
+				"url": "https://registry1:5000",
+				"pollInterval": "6h",
+				"tlsVerify": true,
+				"credentials": {
+					"username": "user",
+					"password": "pass"
+				},
+				"certDir": "/home/user/certs",
+				"content":[
+					{
+						"prefix":"repo1/repo",
+						"tags":{
+							"regex":"4.*",
+							"semver":true
```

Comment:
What happens when semver is false?

Reply:
It will exclude all the tags filtered earlier by "4.*", so right now filtering by tags is an intersection.
Would it be ok if I make it an union?

eg: Sync all tags with regex 4.x and all tags which are not semver.

Reply:
I find it counterintuitive to apply both filters one after the other. @rchincha what do you think?

Reply:
I haven't touch it yet, I will make it a union, if you all agree.